react/index.js

```diff
@@ -81,7 +81,19 @@ class CategoryMenu extends Component {
       departmentsIds.includes(category.id)
     )
 
-    return (departmentsSelected.length && departmentsSelected) || categories
+    const departmentsWithIcons =
+      departmentsSelected.length &&
```

Comment:
Not necessary, since [https://github.com/vtex-apps/category-menu/pull/75/files#diff-8587e609aaa7b1140e400929368c45dfR96](this) checks it later